src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -1,10 +1,12 @@
 import Scroll from 'react-scroll';
 
+export const $ = selectorOrElement =>
```

Comment:
Looks like this is only ever used as a `document.querySelector`. What am I missing?

Reply:
You're right, I'll remove the extra code.

Reply:
oh wait... the `focusOnChange` function calls `focusElement` with a DOM element, which uses this function.